mypy/build.py

```diff
@@ -2336,8 +2336,9 @@ def type_check_second_pass(self) -> bool:
             return False
         t0 = time_ref()
         with self.wrap_context():
-            return self.type_checker().check_second_pass()
-        self.time_spent_us += time_spent_us(t0)
+            result = self.type_checker().check_second_pass()
```

Comment:
I'm not sure if I agree with this refactor. If you want to measure the whole context manager, you should have the addition of `time_spent_us` after it exits, if you want to measure just the time of the second pass, you should put the `t0 =...` line below `self.wrap_context` so that isn't measured.

Reply:
Before that line was simply unreachable, that just didn't make _any_ sense.

Reply:
Anyway, for consistency with other call sites I am going to move this outside of `with`.